docs/source/_toctree.yml

```
@@ -50,6 +50,8 @@
       title: Load audio data
     - local: audio_process
       title: Process audio data
+    - local: audio_dataset_repo
+      title: Create an audio dataset repo
```

```suggestion
      title: Share an audio dataset
```
The `audio_dataset_repo.mdx` can also be renamed `share_audio_dataset.mdx` if we use this title